components/VM_Arm/configurations/vm.h

```diff
@@ -78,6 +78,7 @@
         string initrd_name = "linux-initrd"; \
         string linux_bootcmdline = ""; \
         string linux_stdout = ""; \
+        string dtb_base = ""; \
```

Comment:
maybe `dtb_base_name` or `dtb_name` is a better name choice here (and aligned with the other file name params) , so this can't be confused with a base address.

Reply:
Good idea. Updated to `dtb_base_name`.